src/wallet/api.rs

```diff
@@ -153,6 +154,7 @@ pub(crate) enum DisplayAddressType {
     FidelityBond,
 }
 
+#[cfg(not(feature = "tracker"))]
```

Comment:
These are removed in the split-utxo PR and are not needed anymore. Once rebased on master (after the PR merge) this can be removed.